lib/openstack_taster.rb

```diff
@@ -179,23 +192,35 @@ def not_secure?(instance, username)
         sleep TIMEOUT_SSH_RETRY
         retry
       end
-      error_log(instance.name, e.backtrace)
-      error_log(instance.name, e.message)
+      error_log(instance.logger, 'error', e.backtrace, false, 'Inspec Runner')
+      error_log(instance.logger, 'error', e.message, false, 'Inspec Runner')
       return true # TODO: Don't crash when connection refused
     rescue Exception => e
       puts "Encountered error \"#{e.message}\". Aborting test."
       return true
     end
 
+    error_log( instance.logger, 'info',
+      "Inspec Test Results\n" +
+      runner.report[:controls].map do |test| 
+        "#{test[:status].upcase}: #{test[:code_desc]}\n#{test[:message]}"
+      end.join("\n")
+    )
+
     return runner.report[:controls].any?{|test| test[:status] == "failed"}
   end
 
-  def error_log(filename, message, dup_stdout = false)
+  def error_log(logger, level, message, dup_stdout = false, progname = nil)
```

Comment:
This isn't necessarily used just for the test suites. For instance, with SSH errors. Maybe `context`?